packages/amplify-cli/src/extensions/amplify-helpers/push-resources.ts

```diff
@@ -7,17 +7,18 @@ import {
   IAmplifyResource,
   stateManager,
   ManuallyTimedCodePath,
+  getEnvInfo,
+  getProjectConfig,
+  isValidGraphQLAuthError,
+  handleValidGraphQLAuthError,
```

Comment:
I was looking at usage of this and related TODO. maybe this should remain in `amplify-cli` until TODO is done (or perhaps just do the TODO).

The TODO seems to be : `// TODO PL: this needs to be removed once the api category is using the new amplify error class` 

@lazpavel what do you think?

Reply:
See above comment, my thought is to attempt to make 0 changes to functionality during this move. I want the toolkit move to be as isolated as possible to prevent an unduly massive PR into dev at the end of this, and to minimize side effects.